### DISCUSSION TITLE
鸣潮现在什么水平？一度低于崩坏二

### DISCUSSION THREAD

**Discussion by nzj2004** [share121/inter-knot#4828] [灌水]
现在鸣潮还真不如方舟了[捂脸]，开服蔑视方舟，1.1平视方舟，1.2已经仰视方舟了https://www.zhihu.com/question/660477730/answer/3552919412?utm_psn=1817715165187166208
这个链接是我从1.1开始的鸣潮流水观察，作为鸣潮流水夫人，我看着他一点点死掉，高点和最低点都在下降，很无力的感觉

**Comment by March-777th** [comment]
已经是路边一条了<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >

**Reply by nzj2004** [reply]
现在呢？确实让我刮目相看了，几次迭代进化进步是真的明显

**Reply by lcp1234** [reply]
> 现在呢？确实让我刮目相看了，几次迭代进化进步是真的明显

很难不支持
